frontend/views/pages/Payments.vue

```diff
@@ -224,12 +256,22 @@ export default {
           date: meta.createdDate,
           monthstamp: ISOStringToMonthstamp(meta.createdDate),
           amount: data.amount // TODO: properly display and convert in the correct currency using data.currencyFromTo and data.exchangeRate,
-        }
+        })
+      }
 
-        if (this.filterPayment(p)) {
-          payments.push(p)
+      // DELETE THIS BEFORE MERGE!!!
```

Comment:
It might be good to instead create a script that can be run locally to generate fake payments in the actual system. Then these lines could safely be deleted permanently, and there would be more flexibility in testing out this feature.

Reply:
That approach usually works very well, it's something I have in another project and works great